src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,192 @@
+
+![](resources/images/data_entry_beta/limits-edit-button.png) { width=50% }
+
+Adding, changing, or removing limits requires the correct user privileges.
+
+#### Historical data { #aggregate_data_entry_app.history }
+
+To learn more about a data value, you can see the last twelve values in the *History* section of the data details sidebar. [Make sure the data details sidebar is open](#aggregate_data_entry_app.opening_data_details_sidebar), then open the *History* section
+
+![](resources/images/data_entry_beta/value-history.png) { width=60% }
+
+When working through a data entry form, the *History* section is closed by default to prevent too much network data from being sent for every value.
+
+#### Audit log { #aggregate_data_entry_app.audit_log}
+
+Every data entry cell has an audit log that shows when values were changed and who changed them. To see the audit log, [make sure the data details sidebar is open](#aggregate_data_entry_app.opening_data_details_sidebar), then open the *Audit log* section.
+
+<!-- todo: add image when bug fixes merged -->
+
+When working through a data entry form, the *Audit log* section is closed by default to prevent too much network data from being sent for every value.
+
+
+### Printing { #aggregate_data_entry_app.printing }
+
+To print a data entry form, click the *Options* button in the top bar. From the dropdown menu, you can choose to print the form including cell data values, or an empty form.
+
+![](resources/images/data_entry_beta/print.png) { width=80% }
+
+## Working offline { #aggregate_data_entry_app.working_offline }
+
+You don't need to be connected to the internet or DHIS2 server to enter data. Data entered into forms while you are offline is saved to your local computer. When you reconnect to the internet/server your locally saved data automatically syncs with your DHIS2 server.
+
+To work offline, you need to open the Data Entry (beta) app while you are connected to the internet so the data entry forms can be downloaded and stored on your local computer. Downloading of forms happens automatically in the background.
+
+The badge in the header bar at the top of the screen shows your connection status. If you are not connected to the internet or DHIS2 server, the badge shows *offline*. As well as the offline badge, forms cells you enter data will show the *waiting to sync* status.
+
+![](resources/images/data_entry_beta/offline.png) { width=60% }
+
+
+## Features not supported in the beta version { #aggregate_data_entry_app.unsupported_features }
+
+The **Data Entry (beta) app** is constantly evolving and new features are being added. There are some features that the previous Data Entry app offered that are supported yet:
+- **Custom forms** don't support CSS/JS yet. Custom forms that use CSS/JS won't be loaded. We're learning more about how custom forms are used to build a sustainable, flexible solution.
+- **Multi-organisation unit entry** isn't supported.
```

Comment:
I don't know if we want to advertise all of the features we do not support? With this one in particular, Lars wanted to ask people if they were still using this before deciding if we should implement?

If we are advertising non-supported features, we have also do not support validation checks for compulsory data element operands and field combinations (`fieldCombinationRequired: true` property on a data set)

Reply:
I think the point here was that these (custom forms/multi OU) were explicitly dealt with in sections of the old app documentation.
I think it's fine as is for first push, but we can tweak later if necessary.